### DISCUSSION TITLE
Edit shipment - Shipping Service

### DISCUSSION THREAD

**Discussion by BarnetMichal** [amzn/selling-partner-api-models#3967] [General]
Hello,
I use the endpoint 'POST /orders/v0/orders/{orderId}/shipmentConfirmation' for editing the shipment. Unfortunately I can't set a value to 'Shipping Service' via this endpoint, see screenshot. Is there any way to edit/set this value via sp-api?
![Untitled](https://github.com/amzn/selling-partner-api-models/assets/26611033/d031a268-695b-4f3a-93cd-1d263f3c7bf9)

**Comment by codejo911** [comment]
Make sure you have access to PII and have checked all the restricted roles in your app while registering it.

**Comment by leonardonakagawa** [comment]
Hi @BarnetMichal,

Can you please share how are you handling the body in that case?


Best,

Leonardo Nakagawa
Solutions Architect, Selling Partner API Developer Support

**Reply by BarnetMichal** [reply]
Hi @leonardonakagawa 

here it is.
```
{
    "packageDetail": {
      "packageReferenceId": 4048,
      "carrierCode": "Other",
      "trackingNumber": "1ZF2H1530",
      "orderItems": [
        {
          "orderItemId": "11387902",
          "quantity": 1
        }
      ],
      "shipDate": "2024-11-07T20:05:43Z",
      "shippingMethod": "Other",
      "carrierName": "UPS"
    },
    "marketplaceId": "ATVPDKIKX0DER"
  }
```

 Is the value from shippingMethod = 'Shipping Service' ?

Best regards,
Michal
Koongo Development Team
